src/BuildServer/BuildServerRequests.cs

```diff
@@ -0,0 +1,24 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using Newtonsoft.Json;
+
+namespace Microsoft.Oryx.BuildServer
+{
+    public class BuildServerRequests
+    {
+        [JsonProperty(PropertyName = "command")]
+        public string command { get; set; }
+
+        [JsonProperty(PropertyName = "id")]
+        public string id { get; set; }
+
+        [JsonProperty(PropertyName = "status")]
+        public string? Status { get; set; }
+
+        [JsonProperty(PropertyName = "outputPath")]
+        public string OutputPath { get; set; }
```

Comment:
Do we still want 'outputPath' as a property if it's included in 'command'?

Reply:
This file is actually not used. I'm removing this file. Thanks Stella.